definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -48,5 +48,49 @@
     "Label": "Reason for refusal",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPayment",
+    "Label": "How will payment be made?",
+    "HintText": "How will payment be made?",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "ServiceApplicationPaymentTypeEnum",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PBANumber",
+    "Label": "Enter your account number",
+    "HintText": "Example: PBA0896124",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PBAReference",
```

Comment:
ditto

Reply:
What does Ditto mean?

Reply:
It means it's same as what was mentioned before for a similar situation.
I'm referring to this: https://github.com/hmcts/div-ccd-definitions/pull/437#discussion_r468475440

Reply:
Can you please confirm with Jeremy that this field doesn't already exist. Like the one above, it might be named in a not-very-obvious way.

Reply:
Waiting on response from Jeremy